src/rez/tests/test_shells.py

```diff
@@ -189,21 +231,29 @@ def _execute_code(func, expected_output):
 
             out, _ = p.communicate()
             self.assertEqual(p.returncode, 0)
-            token = '\r\n' if platform_.name == 'windows' else '\n'
+
+            # Powershell and Unix uses \n
```

Comment:
Perhaps it would be better to add a `line_terminator` property to Shell class, rather than hardcoding this behaviour here. I had probably assumed at the time that terminator was platform-dependent and not shell-dependent.

Reply:
I've added a `line_terminator` to the shell API.